src/backend/BackendInterface.h

```diff
@@ -182,7 +182,7 @@ class BackendInterface
      */
 
 public:
-    BackendInterface(clio::Config const& config)
+    BackendInterface([[maybe_unused]] clio::Config const& config)
```

Comment:
remove

Reply:
This parameter is referenced within CassandraBackend.h, where it is fed in as input for config_. config_ faces same issue, but do not know 

![image](https://user-images.githubusercontent.com/47958858/210511157-9598b2a5-c62d-4791-9901-f658aa1217ec.png)

Reply:
Update: Deletion did in fact cause a bug chain to appear. config_ from CassandraBackend.h was necessary in CassandraBackend.cpp. Reverting for now. 
![image](https://user-images.githubusercontent.com/47958858/210512425-c855b354-e0c2-4714-b95a-61a1a86da434.png)

Reply:
> This parameter is referenced within CassandraBackend.h, where it is fed in as input for config_. config_ faces same issue, but could unknowingly create further bugs in the rest of the codebase
> 
> ![image](https://user-images.githubusercontent.com/47958858/210511157-9598b2a5-c62d-4791-9901-f658aa1217ec.png) ![image](https://user-images.githubusercontent.com/47958858/210511801-31e052f5-cfab-4b27-9a9c-abbf28d4e888.png)

All you gotta do, is remove the config parameter from `BackendInterface.h`, along with the `#include` for config.
You could actually completely remove the constructor of `BackendInterface`, because it will become default once config is removed.

The config in `CassandreBackend` is of course there to stay. What you will have to do however is remove the call to `BackendInterface` constructor now that it's a `default constructor` again:
```
 CassandraBackend(boost::asio::io_context& ioc, util::Config const& config, uint32_t ttl)
        : config_(config)
        , ttl_(ttl)
    {
        work_.emplace(ioContext_);
        ioThread_ = std::thread([this]() { ioContext_.run(); });
    }
```